assets/kubernetes/service_account.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ecr-reporting-sa
+  namespace: <namespace>
+  #annotations:
+  #  eks.amazonaws.com/role-arn: arn:aws:iam::<account-id>:role/<role-name>
```

Comment:
I recommend removing commented lines. They may cause confusion. Better to add a line in the readme suggesting the addition of that line if necessary.